forest/shared/src/cli/snapshot_fetch.rs

```diff
@@ -448,6 +563,11 @@ mod test {
             .for_each(|case| assert!(filename_from_url(&Url::try_from(*case).unwrap()).is_err()));
     }
 
+    #[quickcheck]
+    fn test_normalize_filecoin_snapshot_name(filename: String) {
+        assert!(normalize_filecoin_snapshot_name("calibnet", &filename).is_err())
```

Comment:
There's a small chance that the filename will be valid, though. You should just check for panics.

Reply:
done